### ISSUE TITLE
"Error when try to connect to server"

### ISSUE THREAD

**Issue by GHulthe** [meowcat/MSNovelist#9]
![image](https://user-images.githubusercontent.com/119439048/204591287-accc7235-e2b2-43cc-9884-1ac8fb4c334c.png)

We have tried to run the latest docker image for MSnovelist in Ubuntu22.04 and 20.04 and in Docker Desktop in Windows10.
Unfortunately we get the error connecting to server, see below (when in Windows Docker Desktop) 
In Linux the error only says cannot connect to server.

Any suggestions?

**Comment by meowcat** [comment]
Hi,
I can reproduce this. Must be an issue with the old 4.4.29 SIRIUS they keep alive for us. I will ask what is up with the server.

**Comment by GHulthe** [comment]
Many thanks, that is very appreciated! I have tried again now and then. It is still down.

Would it be an option to preprocess the data in 4.4.29 Sirius outside the MSNovelist docker and load those folders from Sirius into MSNovelist? Would I still need the Sirius server to work for that pre-processing?

Regards, Gustaf

**Comment by meowcat** [comment]
They fixed it, at least it works for me. Can you confirm?

**Comment by wittetom** [comment]
Hi meowcat, 

I believe I'm having the same issue as Gustaf - There seems to be a problem accessing the sirius server.  I'm running the latest docker, also in ubuntu.  I get the following error when running the "377.mgf" example: 

WARNING19:28:58 - Could not parse training structure InChI, skipping this entry
WARNING19:29:00 - Error when try to connect to Server. Try again in 4.0s
 Cause: peer not authenticated

On another mgf file I created, I got the following error: 

WARNING19:25:09 - Error when try to connect to Server. Try again in 4.0s
Cause: Could not lookup formula: C26H44O4

(and this was the correct formula predicted, so that part of sirius is functioning as expected). 

Thanks
Tom

**Comment by ZhuYan-SIPPE** [comment]
Hi meowcat,

I have the similar problem as GHulthe，when I running the example data of 377.mgf:

"WARNING08:37:44 - /msnovelist-data/1677487053/spectra-edited.mgf: Set ion to [M + H]+
WARNING08:37:47 - Error when try to connect to Server. Try again in 4.0s
 Cause: Error when querying REST service. Bad Response Code: 404 | Message: Not Found| Content: <html>"

I pulled the lastest docker image of msnovelist in Docker Desktop on Win10.
I tried in Ubuntu 20.04 and Ubuntu22.04, the same error occured.

Could you give me some advice about this?

**Comment by meowcat** [comment]
**Dear everyone,**

at the moment the CSI:FingerID service backing MSNovelist is **down**. We will need to retrain the model for the new service version, which will unfortunately take a while.

Sorry for the inconvenience.

**Comment by Jack-Hu-2001** [comment]
When will the server be back on line again, please?

**Comment by GHulthe** [comment]
Thanks for supporting MSNovelist , we need to have the Sirius server back on-line again.
I try regularly to use MSNovelist without success.

Regards,
Gustaf

**Comment by Jack-Hu-2001** [comment]
Yeah, I've been trying to use MSNovelist every day to see if it's OK in the past 2 weeks. However, I never success in connecting the server.

**Comment by meowcat** [comment]
Hi, sorry this is not possible, as 1) the CSI:FingerID code is not public and 2) as far as I am informed, it requires a lot of resources. However I can inform you at least that we are making some progress in the migration to new Sirius.

**Comment by zorzansi** [comment]
Hi, thank you for the reply. Resources-wise, it could not be an issue as I work on a research institution and we can deal with code requiring high resources. So if point 1 can be tackled in any way (maybe an older open-source version, or some open source alternative which might be interfaced), that might work.
Any possible estimate of the time needed to get the service up again? Days, weeks, months, or more?

**Comment by GHulthe** [comment]
Great to hear!
Thanks,
Gustaf

**Comment by Mitchell-McInerney** [comment]
Hi there

I am really excited to start using MSNovelist but I can see that the server connection issues persist, probably because you guys are still working on this. Can we have any idea of a time frame for bringing MSNovelist back online?

I can get SIRIUS/CSI:Finger ID to work fine on my desktop, is it possible to take pre-processed .mgf files from here and plug them straight into MSNovelist online? This could be a great workaround while you guys sort out the backend stuff. 

Any news/updates would be greatly appreciated!

**Comment by GHulthe** [comment]
Thanks, that is great news!
Do you know if Sirius 6 will be freely open also to MS-researchers within companies?

Regards,
Gustaf

**Comment by meowcat** [comment]
I imagine that it will follow the same licensing as SIRIUS 5, but I have no say in this.
